docs/gdr-s3-data-sharing.md

```diff
@@ -0,0 +1,110 @@
+https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/
+
+In summary:
+1. Go to https://aws.amazon.com/
+2. You clicked on the create an AWS Account button on the top right. 
+  <br/><img src="./screenshot/sign-in-button.png" width="300px"/><br/>
+3. Fill in the form with the information needed. (For support plan, you could choose `Basic support - Free`)
+4. <br/><img src="./screenshot/sign-up-page.png" width="300px"/> <img src="./screenshot/support-plan-page.png" width="300px"/><br/>
+5. You may need to verify your account (Check your email). After you have verified the email, go to your aws console.  
+  <br/><img src="./screenshot/sign-up-complete.png" width="300px"/><br/>
+  You could click on the `Go to the AWS Management Console` or head back to https://aws.amazon.com/ and click `Sign In` button.
+5. Select `Root user` , and fill in the email and password to logged in as a root-user.
+  <br/><img src="./screenshot/sign-in-page.png" width="300px"/><br/>
+
+6. Your AWS account is created! You are now logged in as a root user (user with all access).
+
+
+## Step 2 - Prepare an S3 bucket
+
+**TL;DR**  
+Prepare an `ap-southeast-2` S3 bucket that will store the data.
+
+___
+In AWS, you would need to have permissions in order to create/use/modify services. The services needed for this section is full access to S3 bucket and IAM role permission. If you are in the root-account, you do not need to worry about this.
+
+There is a documentation from AWS and some tutorial provided by AWS. You could have a read through and familiarize with the service.
+https://docs.aws.amazon.com/AmazonS3/latest/userguide/GetStartedWithS3.html
+
+
+### Creating the S3 bucket
+
+1. Search on S3 on the search bar, and open the S3 service console.
+2. Click on the `Create bucket` button.
+3. Enter a Bucket name, that is globally unique. Meaning no other people will have this name even on different AWS account. There is some bucket naming rules that need to be followed ([Check it here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html)).
+4. Select AWS Region to `Asia Pacific (Sydney) ap-southeast-2`. Do **NOT** miss this step.
+5. You leave the rest as default and hit the `Create bucket` button.
+6. Your bucket is created!
+
+### Taking the S3 ARN (Amazon Resource Number)
+In AWS, every resource is labeled with an ID called Amazon Resource Number. We need this so that we could refer this resource in the future.
+
+1. Click on the S3 bucket with the name you created on top. (Could use search bucket name if needed)
+2. On the tabs, click on properties. You could see this arn values. Copy this ARN value
+  <br/><img src="./screenshot/s3-arn.png" width="300px"/><br/>
+
+
+## Step 3 - Permission for the AGHA AWS account to send data to your s3 bucket
+
+**TL;DR**  
+Allow AGHA-Service account to upload to S3.  
+
+AWS AGHA role ARN: `arn:aws:iam::602836945884:role/agha-gdr-s3-data-sharing`
```

Comment:
Perhaps simply allow our account (instead of a role) to access the bucket?
It would make it easier for us as we don't have to assume a specific role to check/manually fix things.

Reply:
Yup, agree! Will replace `role/agha-gdr-s3-data-sharing` with a `*`

Reply:
` "arn:aws:iam::602836945884:root"` should do it.

Reply:
Yup, I shortened it to only specify the account number. [It should behave the same way. ](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-accounts)